openpype/tools/project_manager/project_manager/widgets.py

```diff
@@ -141,16 +144,45 @@ def __init__(self, parent=None, dbcon=None):
         inputs_widget = QtWidgets.QWidget(self)
         project_name_input = QtWidgets.QLineEdit(inputs_widget)
         project_code_input = QtWidgets.QLineEdit(inputs_widget)
+        project_width_input = NumScrollWidget(0, 9999999)
+        project_height_input = NumScrollWidget(0, 9999999)
+        project_fps_input = FloatScrollWidget(1, 9999999, decimals=3, step=1)
+        project_aspect_input = FloatScrollWidget(
+            0, 9999999, decimals=2, step=0.01
+        )
+        project_frame_start_input = NumScrollWidget(-9999999, 9999999)
+        project_frame_end_input = NumScrollWidget(-9999999, 9999999)
+
+        default_project_data = self.get_default_attributes()
+        project_width_input.setValue(default_project_data["resolutionWidth"])
+        project_height_input.setValue(default_project_data["resolutionHeight"])
+        project_fps_input.setValue(default_project_data["fps"])
+        project_aspect_input.setValue(default_project_data["pixelAspect"])
+        project_frame_start_input.setValue(default_project_data["frameStart"])
+        project_frame_end_input.setValue(default_project_data["frameEnd"])
+
         library_project_input = QtWidgets.QCheckBox(inputs_widget)
 
         inputs_layout = QtWidgets.QFormLayout(inputs_widget)
         inputs_layout.setContentsMargins(0, 0, 0, 0)
         inputs_layout.addRow("Project name:", project_name_input)
         inputs_layout.addRow("Project code:", project_code_input)
         inputs_layout.addRow("Library project:", library_project_input)
+        inputs_layout.addRow("Width:", project_width_input)
+        inputs_layout.addRow("Height:", project_height_input)
+        inputs_layout.addRow("FPS:", project_fps_input)
+        inputs_layout.addRow("Aspect:", project_aspect_input)
+        inputs_layout.addRow("Frame Start:", project_frame_start_input)
+        inputs_layout.addRow("Frame End:", project_frame_end_input)
 
         project_name_label = QtWidgets.QLabel(self)
         project_code_label = QtWidgets.QLabel(self)
+        project_width_label = QtWidgets.QLabel(self)
+        project_height_label = QtWidgets.QLabel(self)
+        project_fps_label = QtWidgets.QLabel(self)
+        project_aspect_label = QtWidgets.QLabel(self)
+        project_frame_start_label = QtWidgets.QLabel(self)
+        project_frame_end_label = QtWidgets.QLabel(self)
```

Comment:
You don't need these. Project name and code labels are for warnings about invalid name/code.
```suggestion
```